keylime/db/verifier_db.py

```diff
@@ -57,5 +57,7 @@ class VerifierAllowlist(Base):
     id = Column(Integer, primary_key=True)
     agent = relationship("VerfierMain", back_populates="ima_policy")
     name = Column(String(255), nullable=False)
+    generator = Column(String(256))
```

Comment:
Maybe normalize it is too much, because will create also a new JOIN, but I think that this can be moved to an Integer, and user a Python Enum to represent the set of valid values:

https://docs.python.org/3/library/enum.html

Reply:
I don't oppose the use of `Enum` but I would rather have it on a separate PR then... I will do it